extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
Yeap, what this gentleman said :-). (Or maybe add a 30 minutes tier 
between 5 minutes and 3 hours?)

On 01/26/2016 01:10 PM, Pawel Chojnacki wrote:
> In extensions/wikia/Recirculation/RecirculationController.class.php
> <https://github.com/Wikia/app/pull/9556#discussion_r50900363>:
>
>> @@ -20,6 +21,7 @@ public function index() {
>>  		$posts = $fandomDataService->getPosts( $this->type );
>>
>>  		if ( count( $posts ) > 0 ) {
>> +			$this->response->setCacheValidity( self::CACHE_DURATION );
>
> @hakubo <https://github.com/hakubo> I think @macbre
> <https://github.com/macbre> did the work on Cache standarization actually :)
> Anyway I think WikiaResponse::CACHE_VERY_SHORT (5 min) might be used
> instead of 15 minute cache time
>
> This is the ticket about it:
> https://wikia-inc.atlassian.net/browse/PLATFORM-57
> I don't remember rationale clearly now, but I do remember that we used
> have bazilion different cache expiration times and it was quite hard to
> divine what expiries when, especially coupled with memcache expiration
> periods.
>
> So I think if its possible you could use the standard CACHE_VERY_SHORT.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/9556/files#r50900363>.
>